File: itp1/ITP1_1_D/Main.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		int seconds = sc.nextInt();
		int hour_sec = 60 * 60;
		int minute_sec = 60;
		int h = seconds / hour_sec;
		int m = (seconds % hour_sec) / minute_sec;
		int s = seconds % hour_sec % minute_sec;
		System.out.println(String.valueOf(h) + ":" + String.valueOf(m) + ":" + String.valueOf(s));
		sc.close();
	}
}

File: itp1/ITP1_1_D/main.c
//
//  main.c
//  ITP1_1_D
//
//
//

#include <stdio.h>

int main(int argc, const char * argv[]) {
    int seconds, hour_sec, minute_sec, h, m, s;
    scanf("%d", &seconds);
    minute_sec = 60;
    hour_sec = minute_sec * minute_sec;
    h = seconds / hour_sec;
    m = (seconds % hour_sec) / minute_sec;
    s = seconds % hour_sec % minute_sec;
    
    printf("%d:%d:%d\n", h, m, s);
    return 0;
}